class/class-mainwp-logger.php

```
@@ -980,4 +1022,18 @@ public function show_log_file() { // phpcs:ignore -- NOSONAR - complex.
         fclose( $fh );
     }
+
+        /**
+         * Method hook_module_log_record_inserted()
+         *
+         * @param int   $record_id Log id.
+         * @param array $record Log data.
+         *
+         * @return void
+         */
+    public function hook_module_log_record_inserted( $record_id = false, $record = false ) {
```

PHPMD (2.15.0)

The method hook_module_log_record_inserted has a boolean flag argument $record_id, which is a certain sign of a Single Responsibility Principle violation. (Clean Code Rules)

(BooleanArgumentFlag)